pippo-core/src/main/java/ro/pippo/core/Application.java

```diff
@@ -258,6 +257,56 @@ public Route addRoute(String uriPattern, String requestMethod, RouteHandler rout
         return route;
     }
 
+    /**
+     * It's a shortcut for {@link #addPublicRoute(String)} with parameter <code>"/public"</code>.
+     */
+    public Route addPublicRoute() {
```

Comment:
How about `addPublicResourceRoute`  (and probably add *Resource* to the other methods)?  *addPublicRoute* sounds more like a security/authorization action.